### PR TITLE
update property reference examples

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/5.property-reference.md [hunk 1]
```diff
@@ -59,19 +43,44 @@ There are four built-in properties in the edge:
...
------------------------------------------------------------
++-------------+-------------+--------------+--------------+
+| "player100" | "player101" | 11           | 0            |
++-------------+-------------+--------------+--------------+
+| "player100" | "player102" | 11           | 0            |
++-------------+-------------+--------------+--------------+
 ```
 
-This statement returns all the neighbors of vertex `100` over edge type `follow`, by referencing `follow._src` as the starting vertex ID (which, of course, is `100`) and `follow._dst` as the ending vertex ID.
+This statement returns all the neighbors of vertex `player100` over edge type `follow`, by referencing `follow._src` as the starting vertex ID (which, of course, is `100`) and `follow._dst` as the ending vertex ID.
```
Comment (randomJoe211) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/5.property-reference.md:86]: is `player100`

**Review by randomJoe211 — DISMISSED** [review]

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/5.property-reference.md [hunk 2]
```diff
@@ -1,52 +1,36 @@
...
-## Reference From Edge
-
-### For Property
+### For property
 
-You can use the following syntax to get an edge's property.
+To get properties of an edge, use the following syntax.
 
 ```ngql
 edge_type.edge_prop
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/5.property-reference.md:30]: how about `<edge_type>.<edge_prop>`

**Review by randomJoe211 — DISMISSED** [review]

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/5.property-reference.md [hunk 3]
```diff
@@ -1,52 +1,36 @@
...
+You can refer properties in `WHERE` or `YIELD` syntax.
 
-## Reference From Vertex
+## Reference from vertex
 
-### For Source Vertex
+### For source vertex
 
 ```ngql
 $^.tag_name.prop_name
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/5.property-reference.md:10]: how about `$^.<tag_name>.<prop_name>`

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/5.property-reference.md [hunk 4]
```diff
@@ -1,52 +1,36 @@
...
 ```
 
-`edge_type` is the edge's type, meanwhile `edge_prop` is the property.
-
-For example,
-
-```ngql
-nebula> GO FROM 100 OVER follow YIELD follow.degree;
-```
+The `edge_type` is the type of the edg. The `edge_prop` is the property of the edge.
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/5.property-reference.md:33]: how about The `<edge_type>` ...   The `<edge_prop>` is ...

**Review by whitewum — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**